### DISCUSSION TITLE
wrong result for glyph from importOutlines

### DISCUSSION THREAD

**Discussion by br750** [fontforge/fontforge#4745] [Q&A]
Hello, I'm using a python script on a server, to generate font from directory of SVG. It's almost ok, but i have some letters with strange result, FontForge seem to close the path and connect end point with first.
My SVG it's a simple path, not a close path it come from an Android device, draw letters are record (not a bitmap , but points), send to a php script who transforme it to a svg file(one path ex M11,222L12,225 .....). After a python script is call to generate font. I stuck since many day with this problem, and i can't find the good process or option with   importOutlines.
I give you a sample with the (r) letter with failure  , you can see the svg file , the result ttf and the script that i'm using 

Any help is welcome

the purpose of this application is to give children who have writing problems the possibility of making their own font, that's why i need a cursive font.

Thanks

[sample.zip](https://github.com/fontforge/fontforge/files/6566885/sample.zip)

**Comment by ChiefMikeK** [comment]
## Not an answer to the question but 🏴‍☠️🏁

`¯\_(ツ)_/¯` You might find that this font family is is suitable for your needs
https://github.com/ctrlcctrlv/FRBAmericanCursive 

![!](https://raw.githubusercontent.com/ctrlcctrlv/FRBAmericanCursive/main/specimens/hello.png)

**Reply by br750** [reply]
Thanks
But i need to generate myself each font from svg(from android application) , because each child need his personal font. I near the good result but i have this mistake . I seem like fontforge script close the last point of my path with the first, is there an option that forbiden it in importOutlines
<img width="320" alt="Capture d’écran 2021-06-04 à 14 12 12" src="https://user-images.githubusercontent.com/807159/120799346-e70d8800-c53e-11eb-99d5-ec8152bad760.png">
<img width="555" alt="Capture d’écran 2021-06-04 à 14 18 23" src="https://user-images.githubusercontent.com/807159/120800187-f3deab80-c53f-11eb-917f-f98a201d12cb.png">

**Comment by br750** [comment]
I'm stuck.... need help please
<img width="320" alt="Capture d’écran 2021-06-04 à 14 12 12" src="https://user-images.githubusercontent.com/807159/120799346-e70d8800-c53e-11eb-99d5-ec8152bad760.png">
<img width="555" alt="Capture d’écran 2021-06-04 à 14 18 23" src="https://user-images.githubusercontent.com/807159/120800187-f3deab80-c53f-11eb-917f-f98a201d12cb.png">
